### PR TITLE
Track code coverage with coveralls.io.

### PR DESCRIPTION
**Pull request by ariya**
Note that this is my first attempt on using Coveralls. It may still not work at all.

### CODE REVIEW

**Comment by ariya** [comment]
See https://coveralls.io/r/jquery/esprima for the result of the coverage tracking.

File: package.json [hunk 1]
```diff
@@ -69,6 +70,8 @@
         "check-coverage": "istanbul check-coverage --statement 100 --branch 100 --function 100",
 
         "benchmark": "node test/benchmarks.js",
-        "benchmark-quick": "node test/benchmarks.js quick"
+        "benchmark-quick": "node test/benchmarks.js quick",
+
+        "travis": "npm test && cat ./coverage/lcov.info | coveralls"
```
Comment (michaelficarra) [inline package.json]: I hate unnecessary uses of `cat`.

```sh
npm test && coveralls < ./coverage/lcov.info
```

**Comment by michaelficarra** [comment]
I don't understand what value coveralls adds to a project that enforces 100% coverage. Istanbul is already enforcing 100% coverage during CI. :-1:

File: package.json (hunk 1 above)
Comment (ariya) [inline package.json]: Good point (I thought you like cats.... never mind). Should be fixed now.

**Comment by ariya** [comment]
> I don't understand what value coveralls adds to a project that enforces 100% coverage.

Informational, see #1066 for more details.

**Comment by mikesherov** [comment]
:+1: Immediate feedback to any contributor on pull requests can't hurt.

**Comment by ariya** [comment]
Expect a future pull request from me with regressed code coverage, just to verify that this is working :-)

**Pull request closed without merging**